dj/models/node.py

```diff
@@ -82,15 +110,25 @@ class NodeYAML(TypedDict, total=False):
     tables: Dict[str, List[TableYAML]]
 
 
-class Node(SQLModel, table=True):  # type: ignore
+class NodeBase(SQLModel):
     """
-    A node.
+    A base node.
     """
 
-    id: Optional[int] = Field(default=None, primary_key=True)
     name: str = Field(sa_column=SqlaColumn("name", String, unique=True))
     description: str = ""
+    type: NodeType = Field(sa_column=SqlaColumn(Enum(NodeType)))
+    query: Optional[str] = None
+    mode: NodeMode = NodeMode.PUBLISHED
+    status: NodeStatus = NodeStatus.INVALID
```

Comment:
You said in your PR summary that you added `status` to `Node` and here it's on `NodeBase`. Which side it true?

Reply:
Doh! Thanks for catching this, I had a typo in my force push and pushed to the `status-and-node` branch instead of the `status-and-mode` branch 😂 . Updated to have `status` on `Node`.